### PR TITLE
Grids

### PR DESCRIPTION
**Pull request by raghav6202**
Please review the Grid Component.

The U.I snippet is attached below

image

### CODE REVIEW

File: Components/grids/grids.html [hunk 1]
```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (sonishreyas) [inline Components/grids/grids.html:7]: you can add a meta author tag here later

File: Components/grids/grids.html [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+            <div class="component-heading">Grids</div>
+
+            <p class="info-text">Grids are used in dividing the web page into various sections.</p>
+
+            <div class="component-sub-heading">Grid-Two Items</div>
+
+            <p>The two items are divided in a 1:1 ratio.Use the <b>"grid-two-items"</b> class.
+                <b> (e.g. class="grid-two-items")</b>
+
+
```
Comment (sonishreyas) [inline Components/grids/grids.html:27]: if you don't need the space you can remove it

**Review by sonishreyas — APPROVED** [review]
There is a lot of unnecessary spacing in the code you can remove that if you want so it will look clean. Rest looks good to me.

File: Components/grids/grids.css [hunk 3]
```diff
@@ -1 +1,93 @@
...
+    align-items: center;
+    padding: 00.5rem;
+
+}
+
+.item1
+{
+    background-color: var(--grid-item1-clr);
+
+
```
Comment (VLeads) [inline Components/grids/grids.css]: you can fix indentation.

**Review by VLeads — APPROVED** [review]
Only fixing of indentation is needed. Rest all looks good.

**Pull request merged**